deployment/packer/cac_packer.py

```diff
@@ -80,4 +75,4 @@ def run_packer(machine_type, region, creds, aws_config):
     packer_command.append(packer_template_path)
 
     print('Running Packer Command: {}'.format(' '.join(packer_command)))
-    subprocess.check_call(packer_command, env=env)
+    subprocess.check_call(packer_command, env=os.environ.copy().update({'AWS_PROFILE': aws_profile}))
```

Comment:
This is where `AWS_PROFILE` is explicitly passed into the `check_call` to drive Packer.